### ISSUE TITLE
Storage implants no longer expand the amount of storage slots per implant

### ISSUE THREAD

**Issue by ZAMODA** [tgstation/tgstation#68636]
Reporting client version: 514.1585

## Round ID:
187027

## Testmerges:
- [Removes Internals HUD element](https://www.github.com/tgstation/tgstation/pull/68523)
- [Removes requirements from most midround rulesets, and lowers their minimum population to run, because nobody knows what it means](https://www.github.com/tgstation/tgstation/pull/68571)
- [Refactored fundamental circuit components that have varying inputs. Improvements to the integrated circuit UI](https://www.github.com/tgstation/tgstation/pull/68586)
- [Reworks pAIs](https://www.github.com/tgstation/tgstation/pull/68241)

## Reproduction:
simply inject yourself twice, it allows you to implant yourself but you still only have two slots, instead of the 4 you are supposed to get.

**Comment by antropod** [comment]
I never knew it was a feature of storage implant

**Comment by ZAMODA** [comment]
Yeah, before the storage rework you could get 2 per implant up to a maximum of 6-8 storage slots.

**Comment by timothymtorres** [comment]
> Yeah, before the storage rework you could get 2 per implant up to a maximum of 6-8 storage slots.

It would be a good idea to put it in the storage implant description or the traitor menu panel text.

**Comment by BigGrease** [comment]
i use the storage implant all the time lately and i feel cheated knowing it used to be better

**Comment by timothymtorres** [comment]
Relevant code:

https://github.com/tgstation/tgstation/blob/45d2cdb3187e9b1b5a03364907e67916529735f9/code/datums/components/storage/concrete/implant.dm#L1-L18

https://github.com/tgstation/tgstation/blob/45d2cdb3187e9b1b5a03364907e67916529735f9/code/game/objects/items/implants/implant_storage.dm#L24-L40

https://github.com/tgstation/tgstation/blob/45d2cdb3187e9b1b5a03364907e67916529735f9/code/datums/components/storage/storage.dm#L5-L6

https://github.com/tgstation/tgstation/blob/c1617d7319abb7d891b038d7e1e3765f3d6e9697/code/__DEFINES/dcs/flags.dm#L23-L33

https://github.com/tgstation/tgstation/blob/5f9de41e277a67caeda136fe31be40883581672b/code/datums/components/_component.dm#L429-L447
